### DISCUSSION TITLE
[Feature] Album optimizations (Easier remove from album and more)

### DISCUSSION THREAD

**Discussion by MaroIshiku** [immich-app/immich#12025] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

When adding a picture to an album, show which albums already have this picture included with a checkmark.
And when in "Add to Album"-Dialog, if you click on an album marked with a checkmark (already in this album), it will be removed from this folder.
Another neat feature would be the ability to hide pictures in an albums from timeline.

### Platform

- [x] Server
- [x] Web
- [x] Mobile

**Comment by bo0tzz** [comment]
Great suggestions - but one request per discussion, please. I believe the last one already exists.
